src/ghga_service_commons/api/api.py

```diff
@@ -217,6 +219,29 @@ async def correlation_id_middleware(
         return response
 
 
+async def request_logging_middleware(request: Request, call_next):
+    """Measure and log the amount of time it takes to process the HTTP request."""
+    url = request.url
+    start_time = time.time()
+    response = await call_next(request)
+    duration = round((time.time() - start_time) * 1000)
```

Comment:
```suggestion
    duration = int(round((time.time() - start_time) * 1000))
```

Just to make sure that there is no unnecessary ".0" in the text or JSON representation.